docs/community/integrations/vector_stores.md

```diff
@@ -86,59 +86,103 @@ response = query_engine.query("What did the author do growing up?")
 
 Below we show more examples of how to construct various vector stores we support.
 
-**Elasticsearch**
+**Apache Cassandra®**
 
-First, you can start Elasticsearch either locally or on [Elastic cloud](https://cloud.elastic.co/registration?utm_source=llama-index&utm_content=documentation).
+```python
+from llama_index.vector_stores import CassandraVectorStore
+import cassio
 
-To start Elasticsearch locally with docker, run the following command:
+# For an Astra DB cloud instance:
+cassio.init(database_id="1234abcd-...", token="AstraCS:...")
 
-```bash
-docker run -p 9200:9200 \
-  -e "discovery.type=single-node" \
-  -e "xpack.security.enabled=false" \
-  -e "xpack.security.http.ssl.enabled=false" \
-  -e "xpack.license.self_generated.type=trial" \
-  docker.elastic.co/elasticsearch/elasticsearch:8.9.0
+# For a Cassandra cluster:
+from cassandra.cluster import Cluster
+
+cluster = Cluster(["127.0.0.1"])
+cassio.init(session=cluster.connect(), keyspace="my_keyspace")
+
+# After the above `cassio.init(...)`, create a vector store:
+vector_store = CassandraVectorStore(
+    table="cass_v_table", embedding_dimension=1536
+)
 ```
 
-Then connect and use Elasticsearch as a vector database with LlamaIndex
+**Astra DB**
 
 ```python
-from llama_index.vector_stores import ElasticsearchStore
-
-vector_store = ElasticsearchStore(
-    index_name="llm-project",
-    es_url="http://localhost:9200",
-    # Cloud connection options:
-    # es_cloud_id="<cloud_id>",
-    # es_user="elastic",
-    # es_password="<password>",
+from llama_index.vector_stores import AstraDBVectorStore
+
+astra_db_store = AstraDBVectorStore(
+    token=os.environ["ASTRA_DB_APPLICATION_TOKEN"],  # Your Astra DB token
+    api_endpoint=os.environ[
+        "ASTRA_DB_API_ENDPOINT"
+    ],  # Your Astra DB API endpoint
```

Comment:
Style formatting is a bit unfortunate and makes this seem "long-winded" (not our fault).
There is a dedicated notebook for the details, this is just to give the idea, so why not go with the simpler:
```
astra_db_store = AstraDBVectorStore(
    token="AstraCS:xY3b...",  # Your Astra DB token
    api_endpoint="https://012...abc-us-east1.apps.astra.datastax.com",  # Your Astra DB API endpoint
    collection_name="astra_v_table",  # Table name of your choice
    embedding_dimension=1536,  # Embedding dimension of the embeddings model used
)
```

Reply:
I like that idea. really cleans up the appearance, and like you said, we have the notebook for env variable reading etc